docs/projects/resume-screener-website/getting-started.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Getting Started
+id: getting-started
+description: Setup the project on your local machine
+sidebar_label: Getting Started
+keywords: [astro, install, local, node, approach]
+---
+:::info 
+The TCET Resume Screener page is built using AstroJS, an all-in-one web framework designed for speed.
+:::
+
+### Steps to run the site on your local system: 
+These are the steps you need to follow to get this site on your local system.
+
+### Install Git in your computer
+Follow these steps to install git in your computer.
+1. Go to [https://git-scm.com/downloads](https://git-scm.com/downloads).
+2. Click on Windows. Download should start.
+3. Go to downloads and install the package.
+
+### Clone the repo
+Open Git Bash in any folder and paste the following command
+
+```bash
+git clone https://github.com/tcet-opensource/resume-screener-frontend.git
+```
+
+### Install NodeJs
+
+1. Go to [https://nodejs.org/en/download](https://nodejs.org/en/download)
+2. Select <b>Current</b>.
+3. Download the 64-bit .msi version. Follow the steps and install NodeJS.
+
+:::note
+It is important to have NodeJS in your system
+:::
+
+### Open the folder in VS Code
+1. Install [VS Code](https://code.visualstudio.com/docs/?dv=win32user) if not installed. 
+2. Open Windows Terminal in the folder you have cloned the repo, as done in [step 2](#clone-the-repo).
+
+### Install Important Packages/Dependencies
+
+Install docusaurus
+
+```bash
+npm install docusaurus 
+```
+
+You can make changes to your respective files and changes will be shown once you have saved the file.
+
+### Commands used to run locally 
+
+1. To run the commands, make sure that you have installed Node.JS first.
+2. All commands are run from the root of the project, from a terminal
+
+Here are a set of commands used to run locally:
+
+| **Command** | **Action** |
+| -------- | -------- |
+| `npm install`   | Installs dependencies |
+| `npm run dev` | Starts local dev server at `localhost:3000` |
+| `npm run build` | Build your production site to `./dist/` |
+| `npm run preview` | Preview your build locally, before deploying |
+| `npm run astro ...` | Run CLI commands like `astro add`, `astro check` |
+| `npm run astro --help` | Get help using the Astro CLI |
+
+### Steps to run after a Pull / Merge: 
+
+1. To install all dependencies
```

Comment:
Refer this:
https://opensource.tcetmumbai.in/docs/projects/tnp-website/getting-started